libraries/AP_Stats/AP_Stats.cpp

```diff
@@ -32,10 +32,9 @@ const AP_Param::GroupInfo AP_Stats::var_info[] = {
     AP_GROUPINFO("_RUNTIME",    2, AP_Stats, params.runtime, 0),
 
     // @Param: _RESET
-    // @DisplayName: Reset time
-    // @Description: Seconds since January 1st 2016 (Unix epoch+1451606400) since reset (set to 0 to reset statistics)
+    // @DisplayName: Statistics Reset Time
```

Comment:
I like how we're adding "Statistics" to the param descriptions.  I think in general all parameter descriptions should, if possible, start with the feature or library name.